docs/translating/hindi.md

```diff
@@ -1,11 +1,6 @@
 # Hindi translation style guide (हिन्दी अनुवाद शैली मार्गदर्शक)
 
-Use informal Hindi for translation:
-
-- Informal "you" (_तू_) instead of formal form _आप_. Many top software
-  companies (e.g. Google) use the informal one, because it's much more common in
-  the daily language and avoids making translations look like they were written
-  by machines.
+- Use _आप_ as the second-person pronoun. Don't use तुम or तू.
```

Comment:
```suggestion
- Use _आप_ as the second-person pronoun. Don't use तुम or तू.
  See [chat thread](https://chat.zulip.org/#narrow/stream/58-translation/topic/Hindi.20Translation/near/1762384).
```

(or pull the URL out into a Markdown `[name]: url` line)

That way if in the future someone thinks it should be तुम instead, we'll quickly remember what our thinking was back in 2024 and can start from that basis rather than try to reconstruct it.